components/automate-chef-io/content/docs/scan-jobs.md

```diff
@@ -118,6 +118,12 @@ Whenever we are provided with information about a scannable node, we run a detec
 curl -sSX GET "https://automate-url/api/v0/nodes/rerun/id/d92b0c26-0c9a-4a04-b694-82fa979b2578"
 ```
 
+### How to Add an Environment Value for Project Tagging
```

Comment:
we've been trying to avoid describing this as tagging to end users, instead, we've been using phrases like _assign to a project_, or _move into a project_, depending on the situation.

```suggestion
### How to Add an Environment Value for Project Assignment
```